HM2/HM2.py

```diff
@@ -0,0 +1,82 @@
+from flask import Flask
+import random
+import string
+import csv  
+import re
+
+
+app = Flask(__name__)
+
+
+@app.route("/generate_password")
+def generate_password():
+    """
+    from 10 to 20 chars
+    upper and lower case
+    """
+    # string
+    # ascii_lowercase
+    # ascii_uppercase
+    # int
+    # special symbols
+    # return password
+    
+    letters = list(string.ascii_letters)
+    special_symbols = [symbol for symbol in string.punctuation if symbol not in ("\\", "<", ">")] 
+    print(special_symbols)
+    numbers = [str(num) for num in range(10)]
+    all_symbols = letters + special_symbols + numbers
+
+    min_count_symbols = 10
+    max_count_symbols = 20
+
+    while True:
+        genered_pasword = random.choices(all_symbols, k=random.randint(min_count_symbols, max_count_symbols))
+
+        has_letter = any(symb_pswrd in letters for symb_pswrd in genered_pasword)
+        has_special_symbol = any(symb_pswrd in special_symbols for symb_pswrd in genered_pasword)
+        has_number = any(symb_pswrd in numbers for symb_pswrd in genered_pasword)
+
+        if all([has_letter, has_special_symbol, has_number]):
+            return_pasword_string = "".join(genered_pasword)
```

Comment:
you dont need this var return_pasword_string 
just return "".join(genered_pasword)